app/commands/v2/put_content.rb

```diff
@@ -32,7 +32,8 @@ def call
     private
 
       def fill_out_new_content_item(content_item)
-        clear_draft_items_of_same_locale_and_base_path(content_item, locale, base_path) if base_path_required?
+        clear_draft_items_of_same_locale_and_base_path(
+          content_item, locale, base_path) if content_with_base_path?
```

Comment:
Since this method call is no longer on one line it might be more readable to have an if block